src/utils/validateAvaxBurnedAmountEtna.ts

```diff
@@ -0,0 +1,58 @@
+import type { Context } from '../vms/context/model';
+import {
+  isAddPermissionlessDelegatorTx,
+  isAddPermissionlessValidatorTx,
+  isAddSubnetValidatorTx,
+  isCreateChainTx,
+  isCreateSubnetTx,
+  isPvmBaseTx,
+  isExportTx as isPvmExportTx,
+  isImportTx as isPvmImportTx,
+  isRemoveSubnetValidatorTx,
+  isTransferSubnetOwnershipTx,
+} from '../serializable/pvm';
+import type { UnsignedTx } from '../vms/common';
+import type { FeeState } from '../vms/pvm';
+import { calculateFee } from '../vms/pvm/txs/fee/calculator';
+
+export const validateAvaxBurnedAmountEtna = ({
+  unsignedTx,
+  context,
+  burnedAmount,
+  feeState,
+}: {
+  unsignedTx: UnsignedTx;
+  context: Context;
+  burnedAmount: bigint;
+  feeState: FeeState;
+}): { isValid: boolean; txFee: bigint } => {
+  const tx = unsignedTx.getTx();
+
+  const expectedFee = calculateFee(
+    unsignedTx.getTx(),
+    context.platformFeeConfig.weights,
+    feeState.price < context.platformFeeConfig.minPrice
```

Comment:
Can you confirm with @erictaylor whether this is the correct use of `minPrice`?

Reply:
👌

Reply:
checking with Eric, and hopefully we will get more information from Stephen.  I have extracted out the logic and would pass the fee from wallet-sdks to ajs.

and renamed `evmBaseFee` and `evmFeeTolerance` to `baseFee` and `feeTolerance`, so they make sense to be used by evm and avalanche x/p post-etna.

Reply:
Confirmed with Stephen, we don't need this logic. `feeState.price` will always be greater or equal to min price.

Let's adjust this to `feeState.price`.

Reply:
[updated](https://github.com/ava-labs/avalanchejs/pull/880/commits/7502549ed4fcee63a282678d17b321f8be06e201)